Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -0,0 +1,222 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowServiceAreaView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    /// Tracks whether to add facilities or barriers to map.
+    @State private var selectedGraphicType: SelectedGraphicType = .facilities
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: [
+            model.facilitiesGraphicsOverlay,
+            model.barriersGraphicsOverlay,
+            model.serviceAreaGraphicsOverlay
+        ])
+        .onSingleTapGesture { _, point in
+            model.placeGraphicOnTapLocation(at: point, with: selectedGraphicType)
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Picker("Mode", selection: $selectedGraphicType) {
+                    ForEach(SelectedGraphicType.allCases, id: \.self) {
+                        Text($0.label)
+                    }
+                }
+                .pickerStyle(.segmented)
+                Spacer()
+                Menu {
+                    Slider(value: $model.secondTimeBreak, in: 1...15, step: 1) {
+                        Text("Second: \(Int(model.secondTimeBreak))")
+                    }
+                    Slider(value: $model.firstTimeBreak, in: 1...15, step: 1) {
+                        Text("First: \(Int(model.firstTimeBreak))")
+                    }
```

Comment:
A more appropriate view would be `Stepper`, which allows the time breaks to be `Int` values, which makes more sense:

```swift
Stepper("Second: \(model.secondTimeBreak)", value: $model.secondTimeBreak, in: 1...15)
Stepper("First: \(model.firstTimeBreak)", value: $model.firstTimeBreak, in: 1...15)
```

Also, just to confirm, was it intentional to put the second time break first? If so, why is that?